README.md

```diff
@@ -473,7 +472,7 @@ func main() {
 	// Example for binding JSON ({"user": "manu", "password": "123"})
 	router.POST("/loginJSON", func(c *gin.Context) {
 		var json Login
-		if err := c.ShouldBindJSON(&json); err == nil {
+		if err := c.BindJSON(&json); err == nil {
```

Comment:
Why change to BindJSON?

Reply:
The current API does not have `ShouldBindJSON` method, it has `ShouldBindWith` but it functions differently. In this situation with the new API `BindJSON` should be used.

The GoDoc shows the `ShouldBindJSON` but once you `go get github.com/gin-gonic/gin` it is not inside the `context.go`

Please correct me if I'm wrong.

Reply:
have haved `ShouldBindJSON`